my-profile-webapp/src/main/webapp/my-app/lec/services.js

```diff
@@ -41,6 +41,24 @@ define(['angular'], function(angular) {
              });
         };
         
+        var getEmergencyPhoneNumber = function(){
+            return $http.get('/profile/api/emergencyPhoneNumber/get.json').success(
+                function(data, status){
+                    return data;
+                }).error(function(data, status){
+                    miscService.redirectUser(status, "Get emergency contact info");
```

Comment:
Let me clarify my "if we're continuing to care about the identity of the caller" -- maybe the best move here is to drop the caller parameter or make it optional.